src/main/java/com/knowledgepixels/registry/NanopubPage.java

```diff
@@ -19,8 +19,8 @@
 import com.github.jsonldjava.shaded.com.google.common.base.Charsets;
 import com.mongodb.client.ClientSession;
 
-import eu.ostrzyciel.jelly.core.IoUtils$;
-import eu.ostrzyciel.jelly.core.proto.v1.RdfStreamFrame$;
+import eu.neverblink.jelly.core.utils.IoUtils;
+import eu.neverblink.jelly.core.proto.google.v1.RdfStreamFrame;
```

Comment:
Please import the package (google) or use FQN to avoid confusion.

Reply:
<img width="752" alt="Screenshot 2025-05-25 at 22 26 04" src="https://github.com/user-attachments/assets/610aecf4-c57a-4bf2-b3f8-a281812fcca6" />